test_requirements.txt

```diff
@@ -1,22 +1,26 @@
-pytest-pep8
-codecov
-
-h5py
-requests
+pytest-pep8==1.0.6
+codecov==2.0.22
+h5py==2.10.0
+requests==2.23.0
+statsmodels==0.11.0
+cma==2.7.0
 
-keras
-mxnet
-#torch>=1.2.0
-tensorflow
+#ART supported Frameworks
+tensorflow==2.1.0
+keras==2.3.1
+# tensorflow==1.15.2
+# keras==2.2.5
 tensorflow_addons
-scikit-learn==0.22.1
+
+mxnet==1.6.0
 xgboost==1.0.0
 lightgbm==2.3.1
+#torch>=1.2.0
+torch==1.4.0
```

Comment:
I think we should remove or comment these lines. We are currently installing torch==1.3.1 in .travis.yml for testing on Travis because installing it here has led to errors. The current test runs of this PR have run with torch 1.3.1